### ISSUE TITLE
to sudo or not to sudo is the question

### ISSUE THREAD

**Issue by zombiedk** [kelderek/TPM2-LUKS#4]
Hi

just a note it is not very nice that the script uses sudo
a better way is to detect if the script is run as root

```
if (( $EUID != 0 )); then
    echo "script must run with root privileges"
    echo "sudo $0" 
    exit
fi
```

**Comment by kelderek** [comment]
Thanks! I have resolved this as part of version 1.0.0 that I just released.

**Comment by zombiedk** [comment]
nice
now you just need to remove all the places the code uses sudo
